docs/plans/651-652-spa-elevation-and-caps/01-implementer-runbook.md

```
@@ -0,0 +1,563 @@
+===================================================================================
+
+```bash
+cd /home/mforce/dev/cluckwork
+npm --prefix web run verify:sw   # G4
+git push -u origin feat/651-652-spa-elevation-and-caps
+```
+
+Open the PR with `gh pr create`. **The title is the release note and becomes the squashed commit
+subject**, so it must be a conventional-commit subject. Use exactly:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)